neo/rawio/neuralynxrawio.py

```diff
@@ -65,31 +65,41 @@ def _parse_header(self):
         unit_channels = []
         event_channels = []
 
-        self.ncs_filenames = OrderedDict()  # chan_id: filename
-        self.nse_ntt_filenames = OrderedDict()  # chan_id: filename
+        self.ncs_filenames = OrderedDict()  # uid: filename
```

Comment:
We can do this with a tupple for the key:
self.ncs_filenames = OrderedDict()  # (chan_name, chan_id): filename